pkg/services/librarypanels/models.go

```diff
@@ -37,3 +37,11 @@ type createLibraryPanelCommand struct {
 	Name     string          `json:"name"`
 	Model    json.RawMessage `json:"model"`
 }
+
+// updateLibraryPanelCommand is the command for updating a LibraryPanel
+type updateLibraryPanelCommand struct {
+	FolderID int64           `json:"folderId"`
+	UID      string          `json:"-"`
```

Comment:
This shouldn't be on the command, we should do as for the other *Command types in this package, and pass UID as a separate parameter.

Reply:
Ok, that makes sense. I'll look into that.

Reply:
Fixed